docs/Configuration.md

```diff
@@ -0,0 +1,43 @@
+# Configuring Arc.js
+The default configuration settings for Arc.js can be found in its `config/default.json` file.  Here is a description of the settings:
+
+**defaultVotingMachine**
+  The voting machine used by default by `Dao.new` when creating new DAOs.  Default is "AbsoluteVote".
+
+**gasLimit_deployment**
+Gas sent to transactions when migrating contracts and when creating DAOs that don't use a Unversal Controller.  A number like 4543760.
```

Comment:
why not list the current defaults here? i guess thats dupe, you could also link to defaults.json

Reply:
I didn't give the default gas limit because it changes often.  But yeah, I could link to defaults.json.  I'll do that.